### PR TITLE
Telemetry: change performance timing method to ccount

### PR DESCRIPTION
**Pull request by tobonex**
When it comes to performance, to compare Sof with its reference we should use the same method of timing. Changing timing source to ccount for telemetry and performance monitor.

### CODE REVIEW

**Review by lgirdwood — APPROVED** [review]

**Review by marcinszkudlinski — APPROVED** [review]

File: src/schedule/zephyr_ll.c [hunk 1]
```diff
@@ -138,8 +138,18 @@ static inline enum task_state do_task_run(struct task *task)
...
+#endif
+
 	state = task_run(task);
 
+#ifdef CONFIG_SOF_TELEMETRY
+	const uint32_t current_stamp = (uint32_t)telemetry_timestamp();
+
+	telemetry_update(begin_stamp, current_stamp);
+#endif
+
```
Comment (kv2019i) [inline src/schedule/zephyr_ll.c:152]: So this is called more often after the patch, I guess this is intended in the patch.

File: app/boards/intel_adsp_ace15_mtpm.conf [hunk 2]
```diff
@@ -110,3 +110,4 @@ CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING=y
 CONFIG_COMP_STUBS=y
```
Comment (kv2019i) [inline app/boards/intel_adsp_ace15_mtpm.conf:110]: The git commit prefix is wrong, this patch applies to one Intel platform only. And what about other Intel SOF targets? Shouldn't we have same timer source for all Intel platforms?

**Review by kv2019i — CHANGES_REQUESTED** [review]
Mostly ok, but the board config change needs to be marked better. Please see comment.

File: src/schedule/zephyr_ll.c (hunk 1 above)
Comment (tobonex) [reply src/schedule/zephyr_ll.c:152]: Hm, I intended this to be more aligned with  performance overlay... but this does seem to update the values more often which seems unnecessary. Not sure how it should work. I could delete this one commit for now, as this not a main focus of this PR.

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@kv2019i good for you now ?

**Review by kv2019i — APPROVED** [review]
Ok, let's proceed. The git commit prefixes are still wrong, but this is a problem with other PRs as well, so I won't block because of this. E.g. for the ace15 config file, the history looks like this:

```
55ecc19ab6e5 app: intel_adsp_ace15_mtpm.conf: set DRC as built-in
cfd27fbde0e1 src: convert to a loadable module
9f480375057e llext: automatically calculate module addresses
```

Only first one is correct, so it's not just this PR.

But oh well, you did add the board mentions on the summary lines, so this is already better.
For future, git commits should have a module prefix and this should make clear what is changed in a commit:
```
[area]: [summary of change]

[Commit message body (must be non-empty)]

Signed-off-by: [Your Full Name] <[your.email@address]>
```

**Pull request merged**